nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/lossfunctions/SameDiffLoss.java

```diff
@@ -0,0 +1,156 @@
+package org.nd4j.linalg.lossfunctions;
```

Comment:
Add copyright header

Also run intellij code formatter on whole file.

Also minimize the amount of whitespace, for example:
- 2 blank lines between the end of one method and the javadoc of the next method
- Maximum of 1 blank line between method signature and first code/comment line
- No blank lines between the `return ...` and the `}`
- No blank lines between the end of the javadoc (`*/`) and the start of the method signature (`public ...`)